monero-outreach-docs/translations/fr/merchants/how_to_accept_monero_fr.md

```diff
@@ -0,0 +1,85 @@
+
+En opérant sur du code source libre, le logiciel Monero vous habilite à bâtir votre propre portail de paiement. L'équipe de Monero Integrations a par ailleurs déjà fait tout le travail en créant des librairies et des modules qui s'occupent de tous les aspects importants. Vous trouverez leur code ici: [github.com/monero-integrations](https://github.com/monero-integrations).
+
+Sinon, vous pouvez opter pour des systèmes de paiement par Monero développés par des tiers, qui vous fourniront les logiciels, les applications et les interfaces nécessaires pour simplifier au maximum le démarrage. Ils offrent également des interfaces et outils pour vous permettre de gérer votre compte, ainsi que le soutien technique et le service à la clientèle. Allez voir notre Guide des Services de Traitement du Paiement par Monero afin de vous soutenir dans la sélection de votre solution.
+
+### _LeMonero.com_
+
+Notre café fait fureur et nous avons commencé à vendre nos produits en ligne, en plus d'augmenter le nombre d'emplacements physiques. La bonne nouvelle est que nous n'aurons pas besoin de changer de système de paiement, nous n'avons qu'à l'utiliser plus souvent.
+
+L'intégration du panier lors d'achats en ligne est simple: que vous utilisiez un système de paiement à code source libre ou développé par des tiers, des modules pour WooCommerce, Shopify, et autres sont disponibles et faciles à installer.
+
+Nous pouvons maintenant conclure les transactions automatiquement en monnaie fiduciaire. Avec notre kiosque, nous conservions nos paiement XMR; pour le café, nous liquidions une portion des XMR lorsque le taux nous était favorable. Nos besoins d'encaisse et de flux de trésorerie ont désormais augmenté et il nous faut un mécanisme de conversion systématique de XMR en monnaie fiduciaire, même si cela encours des frais supplémentaires ou un taux de conversion moins favorable.
+
+La plupart des systèmes de paiement par des tiers offrent ce mécanisme d'échange en monnaie fiduciaire, service pour lequel ils chargent des frais. Le montant exacte des frais de transaction peut varier selon le fournisseur de service, mais cela peut certainement en valoir la peine pour des raisons pratiques.
+
+Avec un système de paiement à code source libre, vous pouvez développer la solution qui vous convient le mieux en utilisant l'API de vos sites d'échange préférés et des appels monero-wallet-rpc. Ces notions dépassent la portée de ce guide, mais vous pouvez en lire plus ici: [getmonero.org/resources/developer-guides/wallet-rpc.html](https://www.getmonero.org/resources/developer-guides/wallet-rpc.html).
+
+### _En Savoir Plus_
+
+- Comment accepter des Monero avec le portefeuille officiel Monero: [getmonero.org/get-started/accepting](https://www.getmonero.org/get-started/accepting/)
```

Comment:
We're missing something here. Change the link to french getmonero link: https://www.getmonero.org/fr/get-started/accepting/index.html
[getmonero.org/get-started/accepting](https://www.getmonero.org/fr/get-started/accepting/index.html)

Reply:
Link Changed